runhouse/servers/obj_store.py

```diff
@@ -46,6 +48,30 @@ def set_name(self, servlet_name: str):
             )
             .remote()
         )
+        logger.info("Intializing auth cache as remote actor")
+        self._auth_cache = (
+            ray.remote(AuthCache)
+            .options(
+                name="auth_cache",
+                get_if_exists=True,
+                lifetime="detached",
+                namespace="runhouse",
+            )
+            .remote()
+        )
+
+    @staticmethod
+    def call_auth_cache_method(store, method, *args, **kwargs):
```

Comment:
Probably not necessary - I used a helper method like this for the servlet Actors because at one time some were actors and some weren't (and actually now they're all actors). In this case the AuthCache is always an actor, so we can just call methods on it with get and remote directly.